fedimint-api/src/db/mod.rs

```diff
@@ -105,12 +105,12 @@ impl Database {
             .expect("module decoders already registered");
     }
 
-    // use default decoders if not registered?
+    // uses default decoders if not registered
     fn decoders(&self) -> &ModuleDecoderRegistry {
-        self.0
-            .module_decoders
-            .get()
-            .expect("module decoders not registered")
+        static DEFAULT_MODULES: Lazy<ModuleDecoderRegistry> =
+            Lazy::new(ModuleDecoderRegistry::default);
```

Comment:
Lazy static isn't all that useful here. A default registry doesn't allocate anyway afaik, so is super cheap to create. You could just do `.unwrap_or_default()`.

Reply:
Because we need **&**ModuleDecoderRegistry